spec/features/merchants/destroy_spec.rb

```diff
@@ -25,7 +25,7 @@
       expect(page).to_not have_content("Brian's Bike Shop")
     end
 
-    it "I can't delete a merchant that has orders" do
+    xit "I can't delete a merchant that has orders" do
```

Comment:
@mel-rob @KirbyDD Phil and I talked about this; this test will get fixed in my next PR.